src/main/java/com/github/technus/tectech/thing/metaTileEntity/hatch/GT_MetaTileEntity_Hatch_Param.java

```diff
@@ -157,15 +157,15 @@ public void loadNBTData(NBTTagCompound aNBT) {
                 aNBT.hasKey("eInput1i")){
             boolean usesFloat = aNBT.getBoolean("eFloats");
             if(usesFloat){
```

Comment:
arent these lines 160-163 limited to usesFloat being true? I dont know when that is ever true but there it actually should be integer/float or not?